### DISCUSSION TITLE
OCR system producing no output when run on GPU

### DISCUSSION THREAD

**Discussion by emily-bianchi** [PaddlePaddle/PaddleOCR#12769] [Q&A]
Hi everyone,

I'm trying to run a script to perform OCR (detection+recognition) on GPU but apparently it seems when its run on GPU no output is produced. No bboxes detected and (subsequently) no text/characters recognized. Same result as if it was a blank document.
The exact same script performed on the same document, with exact same parameters works just fine when run on CPU.
So by simply running it on GPU, the result changes.
Below I've attached the terminal output of both runs.
![paddle_ocr_gpu_annotated](https://github.com/PaddlePaddle/PaddleOCR/assets/171232158/1e5cb831-7d21-4e91-9e1f-0aab9b968681)
As you can see, there is no warning nor any other message printed, so I've got no indication of what might potentially be going wrong.
The True in the message "GPU available:True" is the result of the function `is_compiled_with_cuda()` from `paddle.device`.
Finally, if I monitor the GPU usage with `nvidia-smi` when running the script, I can see the GPU being utilized. 

Any idea of what this might be caused by?

Some info about my OS/env that might be helpful.
OS: Ubuntu 22.04
Python: 3.10.13
PaddlePaddle GPU: 2.6.0.post120  || installed with `python -m pip install paddlepaddle-gpu==2.6.0.post120 -f https://www.paddlepaddle.org.cn/whl/linux/mkl/avx/stable.html`
PaddleOCR: 2.7.3
CUDA: 12.4
CuDNN: 8.9.2.26
Nvidia Driver: 550.67

Any help would be greatly appreciated.
Thanks

**Comment by GreatV** [comment]
What kind of gpu are you using? paddle do not support old GPU.

**Comment by emily-bianchi** [comment]
> What kind of gpu are you using? paddle do not support old GPU.

Tesla P40 @GreatV

**Comment by GreatV** [comment]
please try new GPU

**Comment by emily-bianchi** [comment]
So I've tried on a RTX 4060, and in all fairness it worked.
Could you provide more info about which GPUs are supported? Maybe generation/architecture?
Thanks

**Comment by GreatV** [comment]
maybe you can refer: https://www.paddlepaddle.org.cn/documentation/docs/en/install/Tables_en.html
